packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -342,13 +341,17 @@ Content-Type: text/json;charset=UTF-8
 
 <ApiOperation method="patch" url="/Users/${userId}" />
 
-In some situations, you may want to send a PATCH operation to modify user attributes. For more information about modifying PATCH operations, see [RFC 7644, Section 3.5.2](https://tools.ietf.org/html/rfc7644#section-3.5.2). Okta can modify any mapped SCIM attributes through a PATCH operation.
+A user resource can be updated through a PATCH operation for the following actions:
 
-To update users, the functionality needs to be enabled in Okta. In the Okta Admin Console, select the SCIM application from your list of applications. Under the **Provisioning** tab, click **To App**. In the **Update User Attributes** option, click **Enable**.
+* Activating a user
+* Deactivating a user
+* Syncing the user password
 
-One important attribute modified through a PATCH operation is the `active` attribute. This attribute represents the user's current status.
+The `active` attribute in a user profile represents the user’s current status.  
```

Comment:
The apostrophe used in the word "user's" is a character that will confuse our site generator.